### PR TITLE
Case Details / Summary UI update

### PR DESCRIPTION
**Pull request by cleargif**
https://www.pivotaltracker.com/story/show/130010317

### CODE REVIEW

File: app/views/shared/_certification.html.haml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         %th{scope: "col"}
           = t('.instructed_heading')
         %td
-          = claim.certification.certified_by
+          = claim.external_user.name
```
Comment (cleargif) [inline app/views/shared/_certification.html.haml:13]: @stephenrichards @zheileman I need a hand updating the spec for this..

File: app/views/shared/_summary.html.haml [hunk 2]
```diff
@@ -90,11 +90,13 @@
...
               %tr.no-border
                 %td
-                  = expense.date
+                  = expense.date.strftime(Settings.date_format)
                 %td{scope:'row'}
                   = expense.name
                 %td.numeric
                   = expense.distance
+                  - if expense.distance?
+                    = expense.mileage_rate_id == 1 ? '(25p)' : '(45p)'
```
Comment (cleargif) [inline app/views/shared/_summary.html.haml]: @zheileman @stephenrichards 
Is this the best way..?

File: app/views/shared/_message_controls.html.haml [hunk 3]
```diff
@@ -12,7 +12,7 @@
     .grid-row.js-test-send-buttons
       Enter a message below and optionally attach a file (maximum 1 file per message).
       .message-column
-        = f.text_area :body, as: :text, rows: 5, placeholder: t('.message_placeholder'), label: false, class: 'form-control'
+        = f.text_area :body, as: :text, rows: 10, placeholder: t('.message_placeholder'), label: false, class: 'form-control'
```
Comment (zheileman) [inline app/views/shared/_message_controls.html.haml:15]: I think 10 rows might be too big. It hinders the ability of the user to see the feedback of the message sent/failed, as the text area will fill half of the window.
But probably we should also rethink how we show this feedback.

**Pull request merged**